docs/dom-testing-library/api-async.mdx

```diff
@@ -1,13 +1,33 @@
 ---
 id: api-async
-title: Async Utilities
+title: Async Methods
 ---
 
 Several utilities are provided for dealing with asynchronous code. These can be
-useful to wait for an element to appear or disappear in response to an action.
-(See the [guide to testing disappearance](guide-disappearance.mdx).)
+useful to wait for an element to appear or disappear in response to an event,
+user action, timeout, or Promise. (See the
+[guide to testing disappearance](guide-disappearance.mdx).)
 
-All the async utils are built on top of `waitFor`.
+The async methods return Promises, so be sure to use `await` or `.then` when
+calling them.
+
+## `findBy` Queries
+
+`findBy` methods are a combination of `getBy`
+[queries](queries/about.mdx#types-of-queries) and [`waitFor`](#waitfor). They
+accept the waitFor options as the last argument (i.e.
```

Comment:
```suggestion
accept the waitFor options as the last argument (e.g.
```

"e.g." <-> "For example"
"i.e." <-> "In other words"

-- https://www.merriam-webster.com/words-at-play/ie-vs-eg-abbreviation-meaning-usage-difference

Though it seems like we used this before (and we probably use it in unchanged pages as well) so we should probably do this in a follow-up.